server/src/Validation/Auth.validation.js

```diff
@@ -83,6 +86,48 @@ export default class AuthValidation {
     }
     return next();
   }
+
+  /**
+   * @method validateLogin
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   *
+   * @returns {object} status code and error message properties
+   */
+  static validateLogin(req, res, next) {
+    const { email, password } = req.body;
+    const validEmail = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
good job. please create a utility function to validate all emails so that your codebase can be dry.

Reply:
done...